packages/entities/src/lib/add.mutation.ts

```diff
@@ -40,15 +46,16 @@ export function addEntities<
     const { prepend = false, ref = defaultEntitiesRef } = options;
 
     const { entitiesKey, idsKey } = ref!;
+    const idKey = getIdKey<any>(context, ref);
 
     const asArray = coerceArray(entities);
 
     if (!asArray.length) return state;
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
Trying to access properties of undefined variable `process` in browser will throw.